source/account_structure/index.html.md.erb

```diff
@@ -35,6 +28,12 @@ both GOV.UK Pay and the PSP.
 In this setup, payments to services can go to separate bank accounts because
```

Comment:
Would it be helpful to add this into the list of 'you can' things below?

Reply:
Also a good plan - have added (and rejigged the bullet order so it better matches reality).